docs/content/posts/2023-02-27-release-kubernetes-operator-1.4.0.md

```diff
@@ -84,7 +84,7 @@ We rely on users to share feedback so we can improve and make this a very robust
 
 Until now the operator only integrated with the Flink Kubernetes HA mechanism for last-state and other types of application upgrades. 1.4.0 adds support for the Zookeeper HA storage as well.
 
-While Zookeper is a slightly older solution, many users are still using it for HA metadata even in the Kubernetes world.
+While Zookeeper is a slightly older solution, many users are still using it for HA metadata even in the Kubernetes world.
```

Comment:
```suggestion
While ZooKeeper is a slightly older solution, many users are still using it for HA metadata even in the Kubernetes world.
```

This looks like an obvious typo, but the article could be improved by using the correct title case for ZooKeeper everywhere (there is an internal capital K).

Reply:
I found all of ZooKeeper in this document and replaced it!